### PR TITLE
add detail ram info in rpc

### PR DESCRIPTION
**Pull request by lispc**

### CODE REVIEW

File: vm/host/teller.go [hunk 1]
```diff
@@ -162,6 +162,7 @@ func (t *Teller) DoPay(witness string, gasRatio int64) (paidGas *common.Fixed, e
 				return
 			}
 			t.h.db.SetTokenBalance("ram", ramPayer, currentRAM-ram)
+			t.h.db.ChangeUsedRAMInfo(ramPayer, ram)
```
Comment (wangyu0air) [inline vm/host/teller.go]: delta should be -ram?

File: db/wal/log.pb.go [hunk 2]
```diff
@@ -18,7 +16,7 @@ var _ = math.Inf
 // is compatible with the proto package it is being compiled against.
 // A compilation error at this line likely means your copy of the
 // proto package needs to be updated.
-const _ = proto.ProtoPackageIsVersion3 // please upgrade the proto package
+const _ = proto.ProtoPackageIsVersion2 // please upgrade the proto package
```
Comment (wangyu0air) [inline db/wal/log.pb.go]: update proto librarys version

File: core/blockcache/block_cache.pb.go [hunk 3]
```diff
@@ -18,7 +16,7 @@ var _ = math.Inf
 // is compatible with the proto package it is being compiled against.
 // A compilation error at this line likely means your copy of the
 // proto package needs to be updated.
-const _ = proto.ProtoPackageIsVersion3 // please upgrade the proto package
+const _ = proto.ProtoPackageIsVersion2 // please upgrade the proto package
```
Comment (wangyu0air) [inline core/blockcache/block_cache.pb.go]: update proto librarys version

File: vm/host/teller.go (hunk 1 above)
Comment (lispc) [reply vm/host/teller.go]: It is `used` ram. More means more ram has been used

**Pull request merged**